app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx

```diff
@@ -63,6 +66,18 @@ gql`
       id
     }
   }
+
+  query contacts {
+    me {
+      id
+      contacts {
+        id
+        username
+        alias
+        transactionsCount
+      }
+    }
+  }
 `
```

Comment:
there shouldn't be a separate query to the server when there is already one for this screen (this is a basic graphql optimization 101 of reducing the number of query back to the server to reduce latency and therefore provide a better UX)

Reply:
Still getting started with graphql, my bad sorry!
[Fixed here](https://github.com/GaloyMoney/galoy-mobile/pull/2903/commits/98a4d174bec1131a50f9d99420be1f08eba58557#diff-9f28efbc29720d1ca95f1b3446fe21a4ee0ab32c4b0e0ff8e6bf989ebbd4c7c6)